pkg/pillar/cmd/ledmanager/ledmanager.go

```diff
@@ -194,13 +194,13 @@ var mToF = []modelToFuncs{
 		model:       "raspberrypi.rpi.raspberrypi,4-model-b.brcm,bcm2711",
 		initFunc:    InitLedCmd,
 		displayFunc: ExecuteLedCmd,
-		arg:         "led0",
+		arg:         "ACT",
```

Comment:
I think this commit should be done after the other commit that introduces the possibility of list of different names for the same LED...